frontend/src/metabase/collections/containers/FormCollectionPicker/FormCollectionPicker.styled.tsx

```diff
@@ -1,7 +1,7 @@
 import styled from "@emotion/styled";
 
 import ItemPicker from "metabase/containers/ItemPicker";
-import Button from "metabase/core/components/Button";
+import Button from "metabase/core/components/Button/Button";
```

Comment:
nit: the last `/Button` can be removed

Reply:
a2f4114bed2663f7c9faf40ac2ff4c5cb3f70fd3